frontend/src/components/ResultsSection/ResultsSection.js

```diff
@@ -129,7 +129,7 @@ function constructData(){
     data[numItem]['domains'] = domains
     data[numItem]['domains_raw'] = domains_raw
     data[numItem]['rw_indices'] = rw_indices_cleaned
-    numItem +=1;
+     numItem +=1;
```

Comment:
it's unlikely that this space is needed.  It would be useful to run a [linter](https://en.wikipedia.org/wiki/Lint_(software)), likely ESLint, throughout the whole project.